### PR TITLE
DOC-3867: 4.8.6 CorDapp tutorial bugs

### PR DESCRIPTION
**Pull request by AaronForinton**

### CODE REVIEW

File: content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-flows.md [hunk 1]
```diff
@@ -464,15 +464,15 @@ public class RedeemApples {
                     .withRelevancyStatus(Vault.RelevancyStatus.RELEVANT);
             StateAndRef appleStampStateAndRef = getServiceHub().getVaultService().queryBy(AppleStamp.class, inputCriteria).getStates().get(0);
 
-            //Query output basketOfApple
-            QueryCriteria outputCriteria = new QueryCriteria.VaultQueryCriteria()
+            //Query output basketOfApples
```
Comment (michelemarsden) [inline content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-flows.md:467]: Is it: 
```suggestion
            //Query output BasketOfApples
```

**Review by michelemarsden — COMMENTED** [review]
One comment.

File: content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md [hunk 2]
```diff
@@ -26,52 +26,28 @@ Before you start building states, read [Key concepts: States](../../../../../pla
...
 
-3. Run the following command:
 
-   {{< tabs name="tabs-1" >}}
-   {{% tab name="kotlin" %}}
-   ```kotlin
-   git clone https://github.com/corda/cordapp-template-kotlin.git
-   ```
-   {{% /tab %}}
+3. Run the following command:
```
Comment (AaronForinton) [inline content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md:35]: > Instructions for clone either the kotlin or java version but then the tutorial code is only in Java
From [the ticket](https://r3-cev.atlassian.net/browse/DOC-3867) -> I've chosen to take out the kotlin to reflect this

Comment (AnnaK-R3) [reply content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md:35]: Please do not remove link to the Kotlin repo. It's essential that users have access to the Kotlin code. Instead, add a not saying that the following tutorial is for Java only.

File: content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md [hunk 3]
```diff
@@ -67,11 +68,11 @@ The easiest way to write any CorDapp is to start from a template. This ensures t
...
+4. After you clone the CorDapp template, open the `cordapp-template-kotlin` or `cordapp-template-java` in [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+If you don't know how to open a CorDapp in IntelliJ, see the documentation on [Running a sample CorDapp](../../../../../platform/corda/4.8/open-source/tutorial-cordapp.html#step-2-open-the-sample-cordapp-in-intellij-idea).
 
-   If you don't know how to open a CorDapp in IntelliJ, see the documentation on [Running a sample CorDapp](../../../../../platform/corda/4.8/open-source/tutorial-cordapp.html#step-2-open-the-sample-cordapp-in-intellij-idea).
 
-4. [Rename the package](https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_package) to `tutorial`. This changes all instances of the `template` package in the project to `tutorial`.
+5. [Rename the package](https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_package) to `tutorial`. This changes all instances of the `template` package in the project to `tutorial`. At the dropdown menu that appears, choose 'Rename module'.
```
Comment (AnnaK-R3) [inline content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md:75]:
```suggestion
5. [Rename the package](https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_package) to `tutorial`. This changes all instances of the `template` package in the project to `tutorial`. In the drop-down menu that appears, select **Rename module**.
```

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**